pkg/utils/logging.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package utils
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/pingcap/kvproto/pkg/backup"
+	"github.com/pingcap/kvproto/pkg/import_sstpb"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/tidb/kv"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type zapMarshalFileMixIn struct{ *backup.File }
+
+func (file zapMarshalFileMixIn) MarshalLogObject(enc zapcore.ObjectEncoder) error {
+	enc.AddString("name", file.GetName())
+	enc.AddString("CF", file.GetCf())
+	enc.AddString("sha256", hex.EncodeToString(file.GetSha256()))
+	enc.AddString("start key", WrapKey(file.GetStartKey()).String())
+	enc.AddString("end key", WrapKey(file.GetEndKey()).String())
+	enc.AddUint64("start version", file.GetStartVersion())
+	enc.AddUint64("end version", file.GetEndVersion())
+	enc.AddUint64("total kvs", file.GetTotalKvs())
+	enc.AddUint64("total bytes", file.GetTotalBytes())
+	enc.AddUint64("CRC64 xor", file.GetCrc64Xor())
```

Comment:
please make the labels the same as before. i highly doubt the old output contained spaces like `"start key"`.